src/components/DomainSearch/DomainSearch.module.scss

```diff
@@ -0,0 +1,76 @@
+@use '@zero-tech/zui/styles/dimensions';
+@use '@zero-tech/zui/styles/colors';
+
+.Container {
+	.Section {
+		display: flex;
+		justify-content: space-between;
+		gap: 1rem;
+
+		.Input {
+			flex: 1;
+			max-width: 550px;
+
+			:global(.zui-input-wrapper) {
+				gap: 0.25rem;
+
+				:global(.zui-input-enhancer-start) {
+					.StartEnhancer {
+						color: colors.$grey-lighter-2;
+					}
+				}
+
+				:global(.zui-input-enhancer-end) {
+					padding-right: 0.5rem;
+
+					.EndEnhancer {
+						> * > * {
+							margin: unset;
+						}
+					}
+				}
+			}
+
+			input {
+				padding: 0.75rem 0 !important;
+				border-radius: unset !important;
+			}
```

Comment:
This kind of styling should be done in zUI - restyling zUI components is going to lead to inconsistencies (which is what we're trying to avoid by using zUI). Can you let me know why these needed to be restyled directly, and we can look at changing zUI?